test/app-config-sync-feature-release-toggle-definitions-spec.js

```diff
@@ -1,9 +1,9 @@
-var testHelper = require('../node_modules/synctos/etc/test-helper.js');
+var testHelper = require('synctos').testHelper;
 var errorFormatter = testHelper.validationErrorFormatter;
 
 describe('app-config-sync feature release toggle definitions documents definition', function() {
   beforeEach(function() {
-    testHelper.init('build/sync-functions/app-config-sync/sync-function.js');
+    testHelper.initSyncFunction('build/sync-functions/app-config-sync/sync-function.js');
```

Comment:
The `init` function does not exist in the new version of the test-helper module (it does exist in the old, deprecated `etc/test-helper.js` though).